resources/dicts/relationship_events/normal_interactions/trust/increase.json

```
@@ -60,7 +60,7 @@
 			"m_c thinks about how r_c is always reliable.",
 			"m_c notices how r_c is being helpful around camp.",
 			"m_c has faith in r_c.",
-			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/poss}s.",
+			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/poss}.",

```

```suggestion
			"m_c is grateful that r_c has kept a secret of {PRONOUN/m_c/inposs}.",

```
I imagine the "s" is there because the original author was trying to say "hers" or "theirs" and didn't know about the `inposs` tag (his/hers/theirs) rather than the `poss` tag (his/her/their).